### PR TITLE
set up project skeleton, gradle, travis

### LINKED ISSUE #8: use gradle as build tool

### LINKED ISSUE #9: use travis for CI

### LINKED ISSUE #10: use coveralls for code coverage

### LINKED ISSUE #11: badges in README for travis, coveralls, released version ...
add badges to the README for the following:
- travis
- coveralls
- github version badge

### PR DESCRIPTION
**Pull request by ndushay**
@tingulfsen  we did something useful - please merge

closes #8
closes #9
closes #10
closes #11

### CODE REVIEW

**Comment by ndushay** [comment]
Note that 67% coverage refers to 2 out of 3 lines of code that coveralls is counting, one of which is a class declaration and one of which is a closing brace!

File: src/edu/stanford/dlss/WasapiDownloader.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package edu.stanford.dlss;
```
Comment (tingulfsen) [inline src/edu/stanford/dlss/WasapiDownloader.java]: Can we create a web archiving package? E.g. src.edu.stanford.dlss.webarchiving or something like that? Like we did with edu.stanford.dlss.wowza for embed.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+language: java
+
+jdk:
+  - openjdk7
```
Comment (tingulfsen) [inline .travis.yml:4]: Did we check with LOCKSS that OpenJDK and Java version 7 is the best choice?

**Review by tingulfsen — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (ndushay) [reply .travis.yml:4]: OpenJDK 7  is what is currently on was-downloader, and I had the impression that was going to be the java installed on the new VMs.

File: src/edu/stanford/dlss/WasapiDownloader.java (hunk 1 above)
Comment (ndushay) [reply src/edu/stanford/dlss/WasapiDownloader.java]: sure, can do.

**Comment by ndushay** [comment]
changed package name to `edu.stanford.dlss.was` as requested.

**Review by tingulfsen — APPROVED** [review]

**Pull request merged**